src/session.ts

```diff
@@ -55,9 +51,8 @@ export class MemorySessionStorage<T> implements Storage<T> {
 
   async getItem(name: string): Promise<T | undefined> {
     const entry = this.store.get(name)
-    if (entry == null) {
-      return undefined
-    } else if (entry.expires < Date.now()) {
```

Comment:
Redundant `if else`, execution will stop after return in cause of `entry == null`